webui/src/extension-registry-service.ts

```diff
@@ -173,6 +180,36 @@ export class ExtensionRegistryService {
         });
     }
 
+    getNamespaces(): Promise<Namespace[]> {
+        return sendRequest({
+            credentials: true,
+            endpoint: createAbsoluteURL([this.serverUrl, "user", "namespaces"])
+        });
+    }
+
+    getNamespaceMembers(namespace: string): Promise<NamespaceMembership[]> {
+        return sendRequest({
+            credentials: true,
+            endpoint: createAbsoluteURL([this.serverUrl, "user", "members", "namespace", namespace])
```

Comment:
We should use URLs provided by the server where feasible. For example, the `getNamespaces` endpoint could include a property `membersUrl` in its entries so you don't have to construct it yourself in the client. You can do that by adding an optional field to `NamespaceJson`.